activerecord/test/cases/associations/has_many_through_associations_test.rb

```diff
@@ -1082,6 +1084,18 @@ def test_preloading_empty_through_association_via_joins
     assert_equal [], person.posts
   end
 
+  def test_preloading_empty_through_with_polymorphic_source_association
+    owner = Owner.create!(:name => "Rainbow Unicat")
+    pet = Pet.create!(:owner => owner)
+    person = Person.create!(:first_name => "Gaga")
+    treasure = Treasure.create!(:looter => person)
+    non_looted_treasure = Treasure.create!()
+    PetTreasure.create!(:pet => pet, :treasure => treasure, :rainbow_color => "Ultra violet indigo")
+    PetTreasure.create!(:pet => pet, :treasure => non_looted_treasure, :rainbow_color => "Ultra violet indigo")
+
+    assert_equal [person], Owner.where(:name => "Rainbow Unicat").includes(:pets => :persons).first.persons.to_a
+  end
```

Comment:
@vipulnsward ah I thought 1.9 is a =>, seems like rest of the file has =>, so I thought that being consistent is the best. :-)  Ok, let me change it